frontend/src/containers/OnBoarding.js

```diff
@@ -99,18 +105,22 @@ export class OnBoarding extends Component {
 
     return validateSchema(githubForm.attributes, githubSchema)
       .then(() => createGroupFromGithubRepo(payload, attr.token))
-      .then(() => this.setState({step: 5}))
+      .then(() => this.setState({step: 4}))
```

Comment:
Not sure what you mean here, a step has been removed, instead of 5 there are 4 now (3 steps + thank you)